backend/scripts/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Runs blnd-tender tests.
+# All passed parameters go to --test-arguments parameter of the test executable.
+
+# Dependencies:
+# * Postgres server
+# * pg_tmp (http://eradman.com/ephemeralpg/)
+
+# Prerequisites:
+# * Specify your database user name via 'PGUSER' variable; default user name is your OS username.
+#   If you haven't created a database user yet, run
+#   > sudo -u postgres createuser --interactive --pwprompt
+# * If your database user's password is not empty, specify it in .pgpass file
+#   (see https://www.postgresql.org/docs/9.3/libpq-pgpass.html).
+
+set -e -o pipefail
+
+if ! which initdb > /dev/null; then
+    echo -e "Command 'initdb' not found.\nConsider adding '/usr/lib/postgresql/VERSION/bin/' to your PATH."
+    exit 2
+fi
+
+# A temporal server does not live long (60 sec by default), building tests first.
+stack test --no-run-tests
+
+# Running a server in background.
+# Set log level to warning to avoid polluting NOTICE messages in tests.
+conn_str="$(pg_tmp)&options=-c%20client_min_messages\%3DWARNING"
+
+# Executing tests.
+# This thing wraps every argument into double quotes because that's what stack wants :/
+args=( "${@/#/\"}" ); args=( "${args[@]/%/\"}" )
+TEST_PG_CONN_STRING="$conn_str" stack test --no-nix-pure --test-arguments "${args[*]}"
```

Comment:
1. I am not sure if this wrapping is necessary, I never saw such code, can you please check whether it's really necessary and provide an example where it's necessary?
2. What is `--no-nix-pure`? I can't find it in `stack --help` and `stack build --help`. Maybe I am blind.

Reply:
2. https://docs.haskellstack.org/en/stable/nix_integration/#the-nix-shell

Reply:
Do you or anybody else use stack+nix integration? If someone does, okay, let's keep `--no-nix-pure` here, it's not a big deal.
I heard a number of times that stack+nix integration is quite bad, but if it works fine for you or somebody else, you are welcome to use it of course.

Reply:
>use stack+nix integration

I think no) Will remove